modules/qt/hm.nix

```
@@ -0,0 +1,63 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.stylix.targets.qt = {
+    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    iconThemeName = lib.mkOption {
+      description = "Default QT Icons";
+      type = lib.types.str;
+      default = "ePapirus-Dark";
+    };
+  };
+
+  config = lib.mkIf config.stylix.targets.qt.enable (let
+    cfg = config.stylix.targets.qt;
+    kvconfig = config.lib.stylix.colors {
+      template = ./kvconfig.mustache;
+      extension = ".kvconfig";
+    };
+    svg = config.lib.stylix.colors {
+      template = ./kvantum-svg.mustache;
+      extension = "svg";
+    };
+    kvantumPackage = pkgs.runCommandLocal "base16-kvantum" {} ''
+      mkdir -p $out/share/Kvantum/Base16Kvantum
+      cat ${kvconfig} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.kvconfig
+      cat ${svg} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.svg
```

Would an arguably faster and simpler `cp` not work as well:

```suggestion
      cp --dereference "${kvconfig}" "$out/share/Kvantum/Base16Kvantum/Base16Kvantum.kvconfig"
      cp --dereference "${svg}" "$out/share/Kvantum/Base16Kvantum/Base16Kvantum.svg"
```

Or does the data need to be appended?